test/language/expressions/arrow-function/arrow/arrayfunctions_1.js

```diff
@@ -0,0 +1,10 @@
+// Copyright 2015 Microsoft Corporation. All rights reserved.
+// This code is governed by the license found in the LICENSE file.
+
+/*---
+description: arrow function array manipulations
+es6id: 14.2
+---*/
+
+var arr = [2,4,8,256,3*2];
+assert.sameValue(arr.every(v => v%2 === 0), true, "Array.every failed using arrow function as callback for every.");

```

Comment:
Duplicates

- https://github.com/tc39/test262/blob/master/test/language/expressions/arrow-function/cannot-override-this-with-thisArg.js